addons/website/static/src/js/menu/content.js

```diff
@@ -424,6 +425,14 @@ var PagePropertiesDialog = weWidgets.Dialog.extend({
         this.$('.show_group_id').toggleClass('d-none', ev.target.value !== 'restricted_group');
         this.$('#visibility_password').attr('required', ev.target.value === 'password');
     },
+    /**
+     * Library clears the wrong date format so just ignore error
+     *
+     * @private
+     */
+    _onDateTimePickerError: function (ev) {
+        return false;
```

Comment:
I personally don't like `return false;` and prefer to call the right method on `ev` but whatever :wink:

Reply:
I personally have no preference, I hesitated to align on framework which return false or website editor which stopPropagation, I will keep return false